Update and improve introduction in zha.markdown (#28869)
home-assistant/home-assistant.github.io: source/_integrations/zha.markdown

In the case of this ZHA integration, the Zigbee Coordinator is a physical radio adapter or hardware module, that contains a microcontroller that runs the Zigbee stack which the ZHA integration uses to manage and communicate with the Zigbee network and all its devices. The Zigbee Coordinator will act as the central device in your Zigbee network. Its main responsibility from an end-user's point-of-view is the joining of devices to the network and then controlling them. Being the network bridge interface for Zigbee, it maintains the Zigbee network topology and ensures secure and efficient communication between Zigbee devices.

> Once ZHA has been set up and configured with a Zigbee Coordinator radio adapter or module you will be able to directly join/pair any Zigbee Router devices and Zigbee End Devices to the Zigbee network; [regardless of the manufacturer and brand of that Zigbee-based product](#knowing-which-devices-are-supported), (with few [limitations](#limitations)). While backward compatible with devices that use earlier versions of Zigbee Home Automation (ZHA) and Zigbee Light Link (ZLL) protocols, it is generally recommended to buy Zigbee 3.0 (ZB3) compliant devices. Zigbee 3.0 certified devices usually offer better interoperability with more Zigbee gateway solutions out-of-the-box, however [devices that use manufacturer-specific extensions to add non-standard functions and features will still need quirk code in custom device handlers to fully work with ZHA](#how-to-add-support-for-new-and-unsupported-devices).

Reviewer: Terminology like "has been set up and configured" and "Zigbee Coordinator radio adapter or module" is unnecessarily verbose/redundant and hurts readability. Let's stick to simpler and shorter language, there's no need to be 100% precise in an introduction.

One possibility (roughly):

```suggestion
Once ZHA has been set up with a Zigbee radio, you will be able to join Zigbee devices to the network. While most devices will join, not all functionality [will always be supported out of the box](#knowing-which-devices-are-supported). It is generally recommended to buy Zigbee 3.0 devices. Zigbee 3.0 certified devices usually offer better interoperability, however devices that use manufacturer-specific extensions to add non-standard functions and features will still need [device-specific code to fully work with ZHA](#how-to-add-support-for-new-and-unsupported-devices).
```

Author: Updated to at least remove most jargon.

Author: Changed in https://github.com/home-assistant/home-assistant.io/pull/28869/commits/e9e308f0df0a4dc4eb1b27e706b27d7bfa7b7b1d

> Because a Zigbee relies on "mesh networking" technology it depends on its Zigbee Router devices for expanding the network coverage and size. These are always mains-powered devices that will act as Zigbee routers (often wrongly referred to as signal repeaters) within the Zigbee network mesh to extend its range and expand the total amount of devices that can be added. Zigbee End Device devices on the other hand will not expand the range of the network as they do not pass along any signals/messages from other devices, these are usually all battery-operated devices, but not always. You should therefore make sure that your Zigbee network has enough Zigbee Router devices, and always begin by adding those before adding Zigbee End devices when getting started.

Reviewer: There are dedicated signal repeater devices for sale. I don't think the comment is necessary.

```suggestion
Because a Zigbee relies on "mesh networking" technology it depends on its Zigbee Router devices for expanding the network coverage and size. These are always mains-powered devices that will act as Zigbee routers within the Zigbee network mesh to extend its range and expand the total amount of devices that can be added. Zigbee End Device devices on the other hand will not expand the range of the network as they do not pass along any signals/messages from other devices, these are usually all battery-operated devices, but not always. You should therefore make sure that your Zigbee network has enough Zigbee Router devices, and always begin by adding those before adding Zigbee End devices when getting started.
```

Author: Changed in https://github.com/home-assistant/home-assistant.io/pull/28869/commits/e9e308f0df0a4dc4eb1b27e706b27d7bfa7b7b1d

> You will also keep in mind that a Zigbee network needs to have many Zigbee Router devices or else your network mesh coverage will be severely limited due to the short range and poor wall penetration of Zigbee radio signals. In addition, be aware that all Zigbee devices, and especially the Zigbee Coordinator, can be very sensitive to electromagnetic interference (EMI), As such it is highly recommended that you read and follow the best practice tips below about [Zigbee interference avoidance and network range/coverage optimization)](#zigbee-interference-avoidance-and-network-rangecoverage-optimization).

Reviewer: I think this last paragraph is a little redundant, since we already have a section on RF interference.

Author: Think that at least the link should be keept as IMHO extremly important (probably solves more than half of all issues posted in Home Assistant community forum)

Author: Changed in https://github.com/home-assistant/home-assistant.io/pull/28869/commits/e9e308f0df0a4dc4eb1b27e706b27d7bfa7b7b1d

## Introduction

> This ZHA integration is a hardware-independent Zigbee gateway implementation that can replace most proprietary Zigbee gateways/bridges/hubs/controllers. Zigbee is a low-bandwidth communication protocol that relies on using small low-power digital radios to connect compatible devices to local Zigbee wireless private area networks. ZHA will create a single Zigbee network to which you can then pair/join most Zigbee-based devices that are made for home automation and lighting.

Author: Not needed in the introduction I think as should be in the configuration section instead.

> Before installing the ZHA integration in Home Assistant, you need to connect a Zigbee Coordinator radio adapter that will connect to your Zigbee network. Those normally come in the form of a USB dongle that plugs directly into the same computer that is running your Home Assistant installation. The ZHA integration is compatible with many different "Zigbee Coordinator" adapters from various manufacturers. Be sure to [note the recommendations in the respective sections below before buying a Zigbee Coordinator](#compatible-hardware). A Zigbee network always needs to have one Zigbee Coordinator (it can never have more than one), and Zigbee devices can never be connected to more than a single Zigbee network, however, a Zigbee network can have multiple "Zigbee Router" devices and "Zigbee End Device" products.

Author: Had such an explanation before in this PR but where asked to remove it so I did.

> Before installing the ZHA integration in Home Assistant, you need to connect a Zigbee Coordinator radio adapter, that will act as the physical network bridge interface and single point of security controller for your Zigbee network. Those normally come in the form of a USB dongle that plugs directly into the same computer that is running your Home Assistant installation. The ZHA integration is compatible with many different "Zigbee Coordinator" adapters from various manufacturers. Be sure to [note the recommendations in the respective sections below before buying a Zigbee Coordinator](#compatible-hardware). A Zigbee network always needs to have one Zigbee Coordinator (it can never have more than one), and Zigbee devices can never be connected to more than a single Zigbee network, however, a Zigbee network can have multiple "Zigbee Router" devices and "Zigbee End Device" products.

Reviewer: ```suggestion
Before installing the ZHA integration in Home Assistant, you need to connect a Zigbee Coordinator radio adapter that will connect to your Zigbee network. Those normally come in the form of a USB dongle that plugs directly into the same computer that is running your Home Assistant installation. The ZHA integration is compatible with many different "Zigbee Coordinator" adapters from various manufacturers. Be sure to [note the recommendations in the respective sections below before buying a Zigbee Coordinator](#compatible-hardware). A Zigbee network always needs to have one Zigbee Coordinator (it can never have more than one), and Zigbee devices can never be connected to more than a single Zigbee network, however, a Zigbee network can have multiple "Zigbee Router" devices and "Zigbee End Device" products.
```

> Once ZHA has been set up with a Zigbee Coordinator it will automatically create a Zigbee network and you will be able to join/pair any Zigbee Router devices and Zigbee End Devices. With only a few [limitations](#limitations), most devices will join/pair directly regardless of brand and manufacturer. Technically almost all devices that are compliant with the official Zigbee specifications should offer interoperability, though a newer Zigbee Coordinator with support for later firmware often offers better compatibility with both new and older devices. Still, be aware that [all functionality might not always be supported or exposed for every device out-of-the-box](#knowing-which-devices-are-supported) as some devices that use manufacturer-specific extensions to add non-standard functions and features could sometimes need [device-specific code to fully work with ZHA](#how-to-add-support-for-new-and-unsupported-devices).

Author: Not relevant.

> Note that because Zigbee relies on "mesh networking" technology it depends heavily on having [Zigbee Router devices](#using-router-devices-to-add-more-devices) to expand the network coverage and extend its size. These are always mains-powered devices that route messages to other devices that are located close to them within the Zigbee network mesh to improve the range and increase the total amount of devices you can add.  You should therefore make sure that you add many Zigbee Router devices and not just Zigbee End Devices or else its network mesh connection routes will be limited due to the short range and poor wall penetration of Zigbee radio signals. It is highly recommended that you read and follow all the general tips below about [Zigbee interference avoidance and network range/coverage optimization)](#zigbee-interference-avoidance-and-network-rangecoverage-optimization).

Author: I am not going to add a visual diagram to this PR.

> Note that because Zigbee relies on "mesh networking" technology it depends heavily on having [Zigbee Router devices](#using-router-devices-to-add-more-devices) to expand the network coverage and extend its size. These are always mains-powered devices that route messages to other devices that are located close to them within the Zigbee network mesh to improve the range and increase the total amount of devices you can add.  You should therefore make sure that you add many Zigbee Router devices and not just Zigbee End Devices or else its network mesh connection routes will be limited due to the short range and poor wall penetration of Zigbee radio signals. It is highly recommended that you read and follow all the general tips below about [Zigbee interference avoidance and network range/coverage optimization)](#zigbee-interference-avoidance-and-network-rangecoverage-optimization).

Reviewer: You can ignore the bot.

## Introduction

> This ZHA integration is a hardware independent Zigbee gateway implementation that can replace most proprietary Zigbee gateways/bridges/hubs/controllers. It supports pairing/joining devices that use the Zigbee 3.0 standard and earlier Zigbee standards for home automation and lighting. It follows the Zigbee PRO specification, a low-power wireless communication protocol using small digital radios to create a private Zigbee network mesh that can connect compatible devices within a local area.

Reviewer 2: I don't think should mention "Zigbee 3.0", "earlier Zigbee standards", and "Zigbee PRO" directly in the introduction. We probably don't need to at all.

Before installing the ZHA integration in Home Assistant, you need to connect a Zigbee Coordinator radio adapter, that will act as the physical network bridge interface and single point of security controller for your Zigbee network. Those normally come in the form of a USB dongle that plugs directly into the same computer that is running your Home Assistant installation. The ZHA integration is compatible with many different "Zigbee Coordinator" adapters from various manufacturers. Be sure to [note the recommendations in the respective sections below before buying a Zigbee Coordinator](#compatible-hardware). A Zigbee network always needs to have one Zigbee Coordinator (it can never have more than one), and Zigbee devices can never be connected to more than a single Zigbee network, however, a Zigbee network can have multiple "Zigbee Router" devices and "Zigbee End Device" products.

> Once ZHA has been set up with a Zigbee Coordinator it will automatically create a Zigbee network and you will be able to join/pair any Zigbee Router devices and Zigbee End Devices. With only a few [limitations](#limitations), most devices will join/pair directly regardless of brand and manufacturer. It is generally recommended to buy Zigbee 3.0 compliant devices if possible because those offer better interoperability, though a new Zigbee Coordinator is also backward compatible with older devices as well. Still, be aware that [all functionality might not always be supported or exposed for every device out-of-the-box](#knowing-which-devices-are-supported) as some devices that use manufacturer-specific extensions to add non-standard functions and features could sometimes need [device-specific code to fully work with ZHA](#how-to-add-support-for-new-and-unsupported-devices).

Reviewer 2: Again, not sure we need to recommend buying "Zigbee 3.0" devices. It basically means nothing in terms of compatibility now (unfortunately) and most Tuya products are advertised as "Zigbee 3.0", even though many aren't compatible.

> Because a Zigbee relies on "mesh networking" technology it depends heavily on having [Zigbee Router devices](#using-router-devices-to-add-more-devices) to expand the network coverage and extend its size. These are always mains-powered devices that route messages to other devices that are located close to them within the Zigbee network mesh to improve the range and increase the total amount of devices you can add.  You should therefore make sure that you add many Zigbee Router devices and not just Zigbee End Devices or else its network mesh connection routes will be limited due to the short range and poor wall penetration of Zigbee radio signals. It is highly recommended that you read and follow all the general tips below about [Zigbee interference avoidance and network range/coverage optimization)](#zigbee-interference-avoidance-and-network-rangecoverage-optimization).

Reviewer 2: > a Zigbee

> Optionally you can configure Zigbee grouping of lights, switches, and fans (i.e. support for commanding multiple devices as a single group). In addition, you also have the [option to bind some Zigbee devices](#binding-and-unbinding), like binding a button on a remote control directly to a device or group (i.e. define relations between two or more Zigbee devices that will work even when the Zigbee Coordinator and ZHA integration is not available).

Reviewer 2: I'm not sure we need to mention groups and binding in the introduction.

Once ZHA has been set up with a Zigbee Coordinator it will automatically create a Zigbee network and you will be able to join/pair any Zigbee devices. With only a few [limitations](#limitations), most devices will join/pair directly regardless of brand and manufacturer. It is generally recommended to buy Zigbee 3.0 compliant devices if possible, though a new Zigbee Coordinator is also backward compatible with older devices. Zigbee 3.0 certified devices usually offer better interoperability. Still, not all functionality [will always be supported out-of-the-box](#knowing-which-devices-are-supported) and devices that use manufacturer-specific extensions to add non-standard functions and features will need [device-specific code to fully work with ZHA](#how-to-add-support-for-new-and-unsupported-devices).

> The Zigbee Coordinator will act as the central device in your Zigbee network. In the case of this ZHA integration, the Zigbee Coordinator has a microcontroller and runs the Zigbee stack which is used to manage the Zigbee network and communicate with all its devices. Its main responsibility is the joining/pairing of devices to the network and then controlling them. Being the physical network bridge interface for Zigbee, it maintains the Zigbee network topology and ensures secure and efficient communication between Zigbee devices.

Reviewer: I think this paragraph can be removed entirely.

Author: OK removed

Because a Zigbee relies on "mesh networking" technology it depends on its [Zigbee Router devices](#using-router-devices-to-add-more-devices) to expand the network coverage and extend its size. These are always mains-powered devices that route messages to other devices that are located close to them within the Zigbee network mesh to improve the range and increase the total amount of devices you can add.  You should therefore make sure that you have many Zigbee Router devices or else its network mesh coverage will be limited due to the short range and poor wall penetration of Zigbee radio signals.

Zigbee End Device devices will not expand the range of the network as they do not route any messages from other devices. These are usually all battery-operated devices, but not always as some Zigbee End Devices can sometimes be mains-powered devices. Always begin adding Zigbee Router devices before Zigbee End Devices when creating a new Zigbee network. It is highly recommended that you read and follow all the general tips below about [Zigbee interference avoidance and network range/coverage optimization)](#zigbee-interference-avoidance-and-network-rangecoverage-optimization).

> Optionally you can configure Zigbee grouping of lights, switches, and fans (i.e. support for commanding multiple devices as a single group). In addition, you also have the [option to bind some Zigbee devices](#binding-and-unbinding), like a remote, directly to a device or group (i.e. define relations between two or more Zigbee devices that will work even when the Zigbee Coordinator and ZHA integration is not available).

Reviewer: This paragraph too.

Author: Removed, rewritten and restructuring based on feedback
